src/views/Registration/RegistrationDefineBusiness.vue

```diff
@@ -2,16 +2,24 @@
   <div id="registration-define-business">
     <!-- Name -->
     <section class="mt-10">
-      <header id="name-info-header">
+      <header id="name-header">
         <h2>Name</h2>
       </header>
       <v-card flat class="step-container">
         <NameRequestInfo />
-      </v-card>
 
-      <!-- FUTURE: Business Type -->
+        <!-- FUTURE -->
+        <!-- <v-divider class="my-6" /> -->
+        <!-- <BusinessType /> -->
+
+        <v-divider class="my-6" />
 
-      <!-- FUTURE: Nature of Business -->
+        <header id="nature-of-business-header" />
```

Comment:
Empty header so we have something to scroll to (on validation errors).

I didn't want to assign an ID to the NatureOfBusiness component below as that conflicts with whatever ID I assign to it in the component itself.